<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dean_menu</class>
 <widget class="QWidget" name="dean_menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>186</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_v">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>77</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_nu_De">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>110</y>
     <width>32</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>µ = </string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_De">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>176</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_v">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>77</y>
     <width>32</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>v =</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_p">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>11</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nu_De">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>110</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_D">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>44</y>
     <width>32</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>D = </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_De">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>176</y>
     <width>32</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>De = </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_r">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>143</y>
     <width>32</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>r = </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_D">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>44</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_p">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>32</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>p =</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_r">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>143</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>210</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QPushButton" name="calculation_Dean">
    <property name="text">
     <string>Построить график</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Value_delete">
    <property name="text">
     <string>Удалить значения</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
